cli_tools/gce_vm_image_export/main.go

```diff
@@ -24,30 +24,31 @@ import (
-	cloudLogsDisabled    = flag.Bool("disable_cloud_logging", false, "do not stream logs to Cloud Logging.")
-	stdoutLogsDisabled   = flag.Bool("disable_stdout_logging", false, "do not display individual workflow logs on stdout.")
-	labels               = flag.String("labels", "", "List of label KEY=VALUE pairs to add. Keys must start with a lowercase character and contain only hyphens (-), underscores (_), lowercase characters, and numbers. Values must contain only hyphens (-), underscores (_), lowercase characters, and numbers.")
+	clientID              = flag.String(exporter.ClientIDFlagKey, "", "Identifies the client of the exporter, e.g. `gcloud` or `pantheon`.")
+	clientVersion         = flag.String("client_version", "", "Identifies the version of the client of the exporter")
+	destinationURI        = flag.String(exporter.DestinationURIFlagKey, "", "The Google Cloud Storage URI destination for the exported virtual disk file. For example: gs://my-bucket/my-exported-image.vmdk.")
+	sourceImage           = flag.String(exporter.SourceImageFlagKey, "", "Compute Engine image from which to export")
+	format                = flag.String("format", "", "Specify the format to export to, such as vmdk, vhdx, vpc, or qcow2.")
+	project               = flag.String("project", "", "Project to run in, overrides what is set in workflow.")
+	network               = flag.String("network", "", "Name of the network in your project to use for the image export. The network must have access to Google Cloud Storage. If not specified, the network named default is used.")
+	subnet                = flag.String("subnet", "", "Name of the subnetwork in your project to use for the image export. If	the network resource is in legacy mode, do not provide this property. If the network is in auto subnet mode, providing the subnetwork is optional. If the network is in custom subnet mode, then this field should be specified. Zone should be specified if this field is specified.")
+	zone                  = flag.String("zone", "", "Zone of the image to export. The zone in which to do the work of exporting the image. Overrides the default compute/zone property value for this command invocation.")
+	timeout               = flag.String("timeout", "", "Maximum time a build can last before it is failed as TIMEOUT. For example, specifying 2h will fail the process after 2 hours. See $ gcloud topic datetimes for information on duration formats.")
+	scratchBucketGcsPath  = flag.String("scratch_bucket_gcs_path", "", "GCS scratch bucket to use, overrides what is set in workflow.")
+	oauth                 = flag.String("oauth", "", "path to oauth json file, overrides what is set in workflow.")
+	ce                    = flag.String("compute_endpoint_override", "", "API endpoint to override default.")
+	computeServiceAccount = flag.String("compute_service_account", "", "Compute service account to override default.")
```

Comment:
```suggestion
	computeServiceAccount = flag.String("compute_service_account", "", "Compute service account to be used by exporter Virtual Machine.")
```

Reply:
done

Reply:
"When empty, the Compute Engine default service account is used." ?

Reply:
done